sycl/test-e2e/ESIMD/lsc/lsc_local_accessor_block_load_store.cpp

```diff
@@ -0,0 +1,114 @@
+//==- lsc_local_accessor_block_load_store.cpp - DPC++ ESIMD on-device test ==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+//
+// TODO: Enable the test when GPU driver is ready/fixed.
+// XFAIL: opencl || windows || gpu-intel-pvc
```

Comment:
This means we can't merge this, right? I would like to avoid merging functionality with non-working tests.